app/views/issues/evidence/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<h4 class="header-underline">Evidence information</h4>
+<% if @issue.affected.any? %>
+  <% cache ['issue-evidence-tab', @issue, @issue.evidence] do %>
+    <%= render 'issues/evidence/table' %>
+  <% end %>
+<% else %>
```

Comment:
this is inconsistent (I believe) with Nodes#show>Evidence:empty

https://github.com/dradis/dradis-ce/blob/develop/app/views/nodes/show/_evidence.html.erb

Reply:
updated